entity-service-impl/src/main/java/org/hypertrace/entity/service/util/DocStoreConverter.java

```diff
@@ -130,6 +130,8 @@ private static Filter transform(AttributeFilter filter) {
         return transformToEqFilterWithValueListRhs(filter);
       } else if (ATTRIBUTES_LABELS_FIELD_NAME.equals(filter.getName()) && filter.getOperator() == Operator.IN) {
         return transformToOrFilterChainForStrArray(filter);
+      } else if (ATTRIBUTES_LABELS_FIELD_NAME.equals(filter.getName()) && filter.getOperator() == Operator.NOT_IN) {
```

Comment:
Since, we are already adding support, can we also please add the support for 

```
if (ATTRIBUTES_LABELS_FIELD_NAME.equals(filter.getName()) && filter.getOperator() == Operator.NEQ) {
```

if possible?

Reply:
Do we see this use case? Here, `EQ` means all the array elements matches, right? Or one of the values as well?

Reply:
> Here, EQ means all the array elements matches, right? Or one of the values as well?

I think it means any one of the values as well

Reply:
@skjindal93 I added support for this for completeness for `attributes.labels` But, all this logic may need change for Postgres support.